evaluation.py

```diff
@@ -57,11 +57,20 @@ def eval(model, loss_function, test_iter, args) -> Any:
         loss_tot += loss.item()
         if is_vmf_loss:
             pred_embeds = scores[:-1,:,:].reshape(-1, scores.shape[-1])
-            preds = utils.get_nearest_neighbor(
-                pred_embeds,
-                model.decoder.embedding.weight,
-                return_indexes=True,
-            )
+            pred_embeds = scores[:-1,:,:]
+            eval_batch_size = 128 // pred_embeds.shape[0]
```

Comment:
Maybe pass in the eval batch size as a flag? This will vary from setup to setup and using a magic number will make it easier to forgot about.

Reply:
I will get to that eventually

Reply:
It's a quick fix and IMHO could create the kind of bug that we will spend a lot "rediscovering" down the road if we ever change the hyperparams